yarn-project/simulator/src/client/client_execution_context.ts

```diff
@@ -253,7 +253,7 @@ export class ClientExecutionContext extends ViewDataOracle {
    * @param innerNoteHash - The inner note hash of the new note.
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 251]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->TSDoc is missing the new param desc here.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/4500?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->